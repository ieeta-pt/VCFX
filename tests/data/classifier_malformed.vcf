##fileformat=VCFv4.2
##source=VCFX
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	10000	mal1	A	G	100	PASS	.
malformed_line_with_few_columns
chr1	not_a_position	mal2	C	T	80	PASS	.
chr1	30000	mal3	G		50	LowQual	.
chr1	40000	mal4		A	90	PASS	.
chr1	50000	mal5	G	T,	30	LowQual	.
