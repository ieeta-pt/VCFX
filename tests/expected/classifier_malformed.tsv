CHROM	POS	ID	REF	ALT	Classification
chr1	10000	mal1	A	G	SNP
